tests/end_to_end/env_cleanup/rbt.roc

```diff
@@ -0,0 +1,20 @@
+app "build"
+    packages { pf: "../../../Package-Config.roc" }
+    imports [pf.Rbt.{ Rbt, systemTool, Job, job, exec }]
+    provides [init] to pf
+
+init : Rbt
+init =
+    Rbt.init { default: hello }
+
+hello : Job
+hello =
+    job {
+        command: exec (systemTool "bash") [
+            "-c",
+            "echo $(printenv) > out",
```

Comment:
why echo printenv instead of just printenv > out?

Reply:
Because I'm dumb 🙈 
I'll fix it!